src/plugins/maps_dashboards/README.md

```diff
@@ -0,0 +1,22 @@
+# mapsDashboards
```

Comment:
We need a lot of updates here like what is this plugin used for. We should add the links for how to contribute to this project and all.

Reply:
Will add them as the plugin matures : )